angular/src/directives/navigation/ion-router-outlet.ts

```diff
@@ -233,6 +233,12 @@ export class IonRouterOutlet implements OnDestroy, OnInit {
       this.updateActivatedRouteProxy(cmpRef.instance, activatedRoute);
     } else {
       const snapshot = (activatedRoute as any)._futureSnapshot;
+
+      if (snapshot.component) {
+        throw new Error(`[Ionic Error] Standalone components are not currently supported with ion-router-outlet.
```

Comment:
Are we able to make this a warning? Angular will already throw an error and I worry that developers will see this and think they "broke" something in Ionic.